projects/components/src/viewer/code-viewer/code-viewer.component.ts

```diff
@@ -126,4 +154,26 @@ export class CodeViewerComponent implements OnChanges {
   public onSearch(searchText: string): void {
     this.searchText = searchText;
   }
+
+  private observeDomMutations(): void {
+    const codeViewerContentElement = this.element.nativeElement.querySelector('#code-viewer-content') as Node;
```

Comment:
Select via child queries rather than using native API

Reply:
I didn't get this, can you provide an example?
Since I have merged this PR, I can take this in follow-up as well.

Reply:
For example from combo box:
```ts
  @ViewChild('input', { read: ElementRef })
  public readonly input!: ElementRef;
```